draft-law-moq-carp.md

```diff
@@ -121,6 +121,41 @@ to include a new entry, as defined in Table 1 below:
 
 Every Track entry in a CARP catalog MUST declare a "packaging" type value of "cmaf".
 
+## Timeline description
+This specification extends the METADATA element of the timeline track, defined in WARP
+Section 7, in the following four aspects: 
+
+* Specification of a general scheme for metadata signalled through the METADATA element
+  of the timeline track [Ed.Note: This aspect should also be applied to WARP, thus
+  should be moved to WARP later on.]
+* Definition of a namespace for CARP-specific metadata signalled through the METADATA
+  element of the timeline track
+* Addition of metadata signalling of SAP type
+* Addition of metadata signalling of earliest presentation time
+
+
+### General metadata scheme
+When not empty, the string in the METADATA field MUST contain one or more comma
+separated key=value pairs, formatted as Strings as specified in Section 3.3.3 of
+[RFC9651]. For example, the METADATA field can be "key1=1,key2=3268".
```

Comment:
Should you also include an example in which a string value is presented? For example

"key1=1,key2=""hello-world"",key2=3268".